resources/lib/services/library_updater.py

```diff
@@ -44,69 +59,61 @@ def is_idle(self):
             self.startidle = lastidle
         if lastidle < self.startidle:
             self.startidle = 0
-        idletime = lastidle - self.startidle
-        return idletime >= 300
+        return (lastidle - self.startidle) >= 300
```

Comment:
please leave included the variable 'idletime' as original

Reply:
Can be done :)